pages/osx/codesign.md

```diff
@@ -0,0 +1,11 @@
+# codesign
+
+> Sign code for the Mac certificate store.
```

Comment:
Is `for` correct here? Did you mean `with`?

Reply:
Its actually used for proving to the macOS that it's okay to run some code because you're the developer. I'm updating the description; let me know what you think!